easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -105,6 +105,109 @@ def new_ec_method(self, key, *args, **kwargs):
     return new_ec_method
 
 
+def module_is_available(full_mod_name, modtool, avail_modules, hidden):
```

Comment:
we should probably move this inside `robot_find_minimal_toolchain_of_dependency` as an inner function,

or just simply it to this in `robot_find_minimal_toolchain_of_dependency`:

```python
module_exists = (full_mod_name in avail_modules) or (hidden and modtool.exist([full_mod_name])[0])
```